src/components/RadioButtons.tsx

```diff
@@ -19,33 +23,55 @@ type RadioButtonsProps = {
     /** Callback to fire when selecting a radio button */
     onPress: (value: string) => void;
 
+    /** Potential error text provided by a form InputWrapper */
+    errorText?: MaybePhraseKey;
+
     /** Style for radio button */
     radioButtonStyle?: StyleProp<ViewStyle>;
+
+    /** Callback executed when input value changes (same as onPress, but required by FormProvider for the sake of saving drafts) */
+    onInputChange?: (value: string) => void;
+
+    /** The checked value, if you're using this component as a controlled input. */
+    value?: string;
```

Comment:
Coming from https://github.com/Expensify/App/issues/37290. Since `value` is optional, we missed checking if `value` is undefined in cases where the component is not used as a controlled input in order for it to still work as expected without it. More details [here](https://github.com/Expensify/App/issues/37290#issuecomment-1966682179).